packages/commonwealth/client/scripts/controllers/server/threads.ts

```diff
@@ -397,6 +399,37 @@ class ThreadsController {
     });
   }
 
+  public async updateTopic(
```

Comment:
I kind of agree on the intuition here but i feel its cross cutting. Hitting topics api from topic controller/modal feels safe and for side effects, they could be dispatched to relevant controllers. In scenarios where we need to update multiple controllers it would get out of hand.

Reply:
tbh this should be involved in the threads refactor and updateTopic function should be incorporated into the editThread in my opinion because as I said, this is thread resource modification, not topic. cc @rbennettcw

Reply:
@jnaviask I think we can handle this in a followup when thread api is refactored, we can merge /updateTopic into the new route for edit thread?